sdk/runanywhere-python/runanywhere/_generated_defaults.py

```
@@ -0,0 +1,87 @@
+
+    MAX_TOKENS: Final[int] = 512
+    TEMPERATURE: Final[float] = 0.0
+
+
+class StorageDefaults:
+    """Generated from StorageDefaults in idl/sdk_defaults.proto."""
+
+    CONTEXT_LENGTH: Final[int] = 2048
+
+
+__all__ = ["NetworkDefaults", "AudioCaptureDefaults", "VoiceAgentDefaults", "HybridDefaults", "WorkerDefaults", "FFIDefaults", "EnvironmentDefaults", "StructuredOutputDefaults", "StorageDefaults"]
```

Ruff (0.15.21)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)